src/utils/buffer.ts

```diff
@@ -81,7 +80,10 @@ export const unpackToInt = (n: number, signed: boolean, buff: Buffer): [number,
     }
   } else if (n === 8) {
     // We have only signed ints here
-    if (parseInt(nodeMajorVersion) > 8) {
+    const nodeMajorVersion = get(process, 'versions.node');
+    // Browsers don't have access to versions.node and they don't have readBigInt64BE method
+    // slicedBuff.readIntBE(0, 8) worked fine testing on electron and react native (desktop and mobile wallets)
+    if (nodeMajorVersion && parseInt(nodeMajorVersion.split('.')[0]) > 8) {
```

Comment:
Why don't we just check whether `readBigInt64BE` exists? For example:

```js
if (slicedBuff.readBigInt64BE) {
  retInt = Number(slicedBuff.readBigInt64BE());
} else {
  retInt = slicedBuff.readIntBE(0, 8);
}
```

Reply:
Done